feat(Paragraph): New `noTopMargin` and `noBottomMargin` props that allow customization of which margin to remove. (#1177)
cengage/react-magma: .changeset/feat-paragraphMargins.md

---
'react-magma-dom': minor
---

> feat(Paragraph): Adding more configuration for the Paragraph component margins. `noTopMargin` and `noBottomMargin` both allow a removal of the margin from one side.

Reviewer: Just a nitpick, but the changeset descriptions are what adopters will see when upgrading versions. I think we can be more direct. Instead of `configurations` we should call them `props`.

```suggestion
feat(Paragraph): New `noTopMargin` and `noBottomMargin` props that allow customization of which margin to remove.
```

Also for organization purposes, we should use the same line here as the pull request title. This would allow us to find the pull request in the history quickly and match it to that item in the change log.
